lib/ts/BufferWriter.h

```diff
@@ -32,17 +33,36 @@
 
 namespace ts
 {
-// Abstract class.
-//
+/** Base (abstract) class for concrete buffer writers.
+ */
```

Comment:
Are retro C comments all the fashion rage now?

Reply:
This is the only way to have multi-line comments. When I edit these kind of Doxygen comments not having comment leading is much more convenient.